### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by Coeur**
CLA: trivial

##### Checklist
- [x] documentation is added or updated

### CODE REVIEW

File: crypto/md5/asm/md5-ia64.S [hunk 1]
```diff
@@ -189,9 +189,9 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 //		Reads an input block, then calls the digest block
 //		subroutine and adds the results to the accumulated
 //		digest.  It allocates 32 outs which the subroutine
-//		uses as it's inputs and rotating
-//		registers. Initializes the round constant pointer and
-//		takes care of saving/restoring ar.lc
+//		uses as its inputs and rotating registers.
+//		Initializes the round constant pointer and takes
```
Comment (mspncp) [inline crypto/md5/asm/md5-ia64.S]: _Note: 'takes' is unchanged, except for whitespace changes around it. Seems to be a display error._

**Comment by mspncp** [comment]
The coverage/coveralls failure can safely be ignored.

**Comment by mspncp** [comment]
Oh, I overlooked that your pull request targets the OpenSSL_1_1_0-stable stable branch. This branch will be end-of-life on 2019-09-11 and receives only security updates. 

If you want, I can try to apply your fixes to master and 1.1.1. Or would you prefer to retarget your changes yourself?

**Comment by Coeur** [comment]
I'll rebase for 1.1.1 myself, no worry.

**Comment by mspncp** [comment]
Please do it against master, we‘ll cherry-pick to 1.1.1.

File: crypto/md5/asm/md5-ia64.S (hunk 1 above)
Comment (Coeur) [reply crypto/md5/asm/md5-ia64.S]: md5-ia64.S doesn't exist anymore on 1.1.1 anyway

**Comment by mspncp** [comment]
Well, never mind. 1.1.1 is ok, too.

**Comment by Coeur** [comment]
I've done it against 1.1.1 now. You sure you want it against master?

**Comment by mspncp** [comment]
Closing and reopening tobkick the CI bots.

**Comment by mspncp** [comment]
> Agreeing that these changes are trivial.

Thanks for the reminder :-)  I also agree that it's a trivial change.

**Comment by levitte** [comment]
Same here, I agree this is trivial

**Comment by mspncp** [comment]
Merged to master and 1.1.1, thank you again.

_(Note: the fix for `util/mkdef.pl` was dropped when merging to master. The entire line was gone, no typo anymore.)_

**Pull request closed without merging**